Makefile

```diff
@@ -85,6 +85,10 @@ eslint-branch: node_modules/eslint node_modules/eslint-plugin-react node_modules
 i18n-lint:
 	@echo "$(JS_FILES)" | sed 's/\([^ ]*\/test\/[^ ]* *\)//g' | xargs -n1 $(I18NLINT)
 
+# Skip files that are auto-generated
+mixedindentlint: node_modules/mixedindentlint
+	@echo "$(JS_FILES)\n$(SASS_FILES)" | xargs $(NODE_BIN)/mixedindentlint --ignore-comments --exclude="client/config/index.js" --exclude="shared/components/gridicon/index.jsx"
```

Comment:
I interpreted this as acknowledging that mixed indentation exists in these generated files and therefore we exempt them, when it seems like we should just update the generated output to not include mixed indentation.